pkg/alertmanager/multitenant.go

```diff
@@ -154,6 +161,8 @@ func (cfg *MultitenantAlertmanagerConfig) RegisterFlags(f *flag.FlagSet) {
 
 	cfg.AlertmanagerClient.RegisterFlagsWithPrefix("alertmanager.alertmanager-client", f)
 
+	f.DurationVar(&cfg.PersistInterval, "alertmanager.persist-interval", 15*time.Minute, "The interval between persisting the current alertmanager state (notification log and silences) to object storage. This state is read when all replicas for a shard have failed. In this scenario, having persisted the state more frequently will result in potentially fewer lost silences, and fewer duplicate notifications.")
```

Comment:
Please explicitly mention this is used only when alertmanager sharding is enabled.